rhel-hpc-7_9-install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+lvextend -L +5G /dev/mapper/rootvg-optlv
+lvextend -L +5G /dev/mapper/rootvg-usrlv
+lvextend -L +8G /dev/mapper/rootvg-tmplv
+
+xfs_growfs /dev/mapper/rootvg-optlv
+xfs_growfs /dev/mapper/rootvg-usrlv
+xfs_growfs /dev/mapper/rootvg-tmplv
+
```

Comment:
Are the lines 4-10 not image version specific? is the RHEL image guaranteed to be built as such?